README.md

```diff
@@ -37,6 +37,7 @@ There is also a scheduled workflow called [Retrieve image tags](https://github.c
 
 - `github-releases`: This will use GitHub releases as source, excluding pre-releases. This can be used if you need to keep all tags from the configured images in sync with GitHub releases
 - `github-latest-release`: This will use the release on GitHub marked as `Latest release`. This can be used if you only want one release to be added that is marked as latest.
+- `github-tagged-images-file`: This will look up GitHub releases, excluding pre-releases, and find the list of images inside a specified file in the source code at the release's tag. This can be used if your project maintains a list of images in a file, e.g., https://github.com/longhorn/longhorn/blob/master/deploy/longhorn-images.txt
```

Comment:
```suggestion
- `github-tagged-images-file`: This will look up GitHub git repository tags, and find the list of images inside a specified file. The tag must have an associated release, with the pre-release flag unset. This can be used if your project maintains a list of images in a file, e.g., https://github.com/longhorn/longhorn/blob/master/deploy/longhorn-images.txt
```

Reply:
Thank you. Fixed